ui/app/controllers/vault/cluster/settings/configure-secret-backend.js

```diff
@@ -57,7 +57,6 @@ export default Controller.extend(CONFIG_ATTRS, {
           },
         })
         .then(() => {
-          this.model.send('pushedData');
```

Comment:
This is not a method anymore and was throwing an error so I removed it and have not seen any negative side effects.